spec/controllers/holds_controller_spec.rb

```diff
@@ -34,8 +36,15 @@
       allow(mock_patron).to receive(:holds).and_return(holds)
     end
 
+    it 'sends the right item details to the web service' do
+      item_details = controller.send(:item_details)
+
+      expect(item_details).to eq holdRecordList: true
```

Comment:
👌

Reply:
Don't love this actually, but I couldn't figure out a better way to test that private method. Just feel like that line is important to cover.

Reply:
per our conversation with Ruth, there should be always a call number in the response, so we are good 👍

Reply:
even if in the future there's a case where it doesnt, it would degrade pretty ok (just an empty parenthetical at the end of the title)